BE/FunctionalEntities/FunctionalEntities.rdf

```diff
@@ -132,21 +111,113 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>functional entity</rdfs:label>
-		<skos:definition>any independent party (i.e., person, business entity, governmental entity, or other organization) defined in terms of its function</skos:definition>
+		<skos:definition>independent party defined in terms of its function</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-fct-fct;Merchant">
-		<rdfs:subClassOf rdf:resource="&fibo-be-fct-fct;Business"/>
+		<rdfs:subClassOf rdf:resource="&fibo-be-fct-fct;FunctionalBusinessEntity"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isCharacterizedBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-be-fct-fct;Commerce"/>
+				<owl:onProperty rdf:resource="&lcc-cr;isClassifiedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-be-fct-fct;MerchantCategoryCode"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;isIdentifiedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-be-fct-fct;MerchantIdentifier"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>merchant</rdfs:label>
-		<skos:definition>a business entity engaged in a trading activity</skos:definition>
+		<skos:definition>party engaged in the purchase and sales of goods for profit</skos:definition>
```

Comment:
profit is not required, either by intention or in actuality.

Is a farmer who sells their own crops at a market a merchant by this definition (no goods purchased)?
What about someone who employs builders and builds houses for a profit (there are no goods involved)?